task/sealights-go-oci-ta/0.1/sealights-go-oci-ta.yaml

```diff
@@ -138,17 +148,29 @@ spec:
           TEST_EVENT="pull_request"
         fi
 
+        if ! slcli config init --lang go --token "${SEALIGHTS_TOKEN}"; then
+          echo -e "[ERROR] Failed to initialize Sealights configuration."
+        fi
+
         if [ -n "${TEST_EVENT}" ] && [ "${TEST_EVENT}" = "pull_request" ]; then
           echo "[INFO] Creating build session id from a ${TEST_EVENT} event source code: [ component: ${COMPONENT}, target-branch: ${TARGET_BRANCH}, pull-request-number ${PULL_REQUEST_NUMBER}, latest-commit: ${REVISION} ]"
-          slcli config create-pr-bsid --app "${COMPONENT}" --target-branch "${TARGET_BRANCH}" \
-            --pull-request-number "${PULL_REQUEST_NUMBER}" --latest-commit "${REVISION}" --repository-url "${REPOSITORY_URL}"
+          if ! slcli config create-pr-bsid --app "${COMPONENT}" --target-branch "${TARGET_BRANCH}" \
+            --pull-request-number "${PULL_REQUEST_NUMBER}" --latest-commit "${REVISION}" --repository-url "${REPOSITORY_URL}"; then
```

Comment:
we need to add `--fail-build` flag here, otherwise the `slcli config` command will end with `0` exit code